addons/mail/static/src/components/discuss/tests/discuss_tests.js

```diff
@@ -559,6 +559,28 @@ QUnit.test('sidebar: channel rendering with needaction counter', async function
     );
 });
 
+QUnit.test('sidebar: channel rendering when mailing channel', async function (assert) {
+    assert.expect(1);
+
+    Object.assign(this.data.initMessaging, {
+        channel_slots: {
+            channel_channel: [{
+                channel_type: "channel",
+                id: 20,
+                mass_mailing: true,
+                name: "General",
+            }],
+        },
+    });
+    await this.start();
+    const channel = document.querySelector(`.o_DiscussSidebar_groupChannel .o_DiscussSidebar_item`);
+    assert.containsOnce(
+        channel,
+        '.fa.fa-envelope-o',
+        "should have an icon to indicate that the channel is a mailing channel"
```

Comment:
This is actually not updating correctly (until refresh) when changing an existing channel to mailing (due to the update of channel not triggering bus notif?).

Reply:
Indeed but it's also not the case in production, I thought it was working as intended ?

Reply:
This problem exists for a long time, at least v12 when it introduces gear icon on sidebar items.
Much bigger problem than it looks, and we can't fix it without some changes in python.
So definitely out-of-scope of PR.

Reply:
Yes, just noting it somewhere as I tested it. Should probably create a task.

Reply:
subtask of refactoring or new independant task ?

Reply:
Would put in backlog subtask, because discuss tasks need to be cleaned again while backlog column is less of a mess.
But honestly, I don't really care, because this is a problem I still have in mind regardless where we leave a mark.

Reply:
https://www.odoo.com/web#id=2286018&action=3206&active_id=1519&model=project.task&view_type=form&cids=1&menu_id=4720